src/manifest.json

```diff
@@ -10,14 +10,10 @@
   "content_scripts": [
     {
       "run_at": "document_start",
-      "matches": [
-        "https://github.com/*/*/pulls*",
-        "https://github.com/*/*/pull/*",
-        "https://github.com/*/*/issues*",
-        "https://github.com/*/*/discussions*"
-      ],
-      "css": ["src/github/orbit-action.css"],
-      "js": ["src/github/background.js"]
+      "matches": ["https://github.com/*"],
+      "exclude_matches": ["https://*/login/*"],
```

Comment:
Am I right in thinking these are the only changes related to this actual PR, the rest is just bundled in from the hot reloading?

Reply:
You are correct! I thought I instructed GitHub to only show the specific commit (by changing the base branch), but maybe a force push threw it off tracks. I’ll rebase, it should fix this.